### DISCUSSION TITLE
Add a method to convert sdr to dolby vision directly like with unifab

### DISCUSSION THREAD

**Discussion by pitrako** [R3S3t9999/DoVi_Scripts#38] [Ideas]
Add sdr to dolby vision direct conversion.

**Comment by R3S3t9999** [comment]
The script can put SDR into a 100nits BT2020 container(workflow 8-2-5) but it's useless without [manually grading it brighter](https://youtu.be/728R3ppSWuk) and I don't believe in any one-click SDR to HDR/DV converter. So no sorry, this wont happen in DoVi_Scripts.
